empress/topoSort/test_ReconBuilder.py

```diff
@@ -0,0 +1,150 @@
+    """
+    @classmethod
+    def setUp(self):
+        """
+        generate newick tests of certain sizes
+        """
+        size_range = [5]
+        generated_dir_path = "./newickSample"
+        generateNewickTestsMultipleSizes(size_range, generated_dir_path)
+
+    @staticmethod
+    def build_ordering_dict(host_tree, parasite_tree, reconciliation):
+        """
+        build the ordering dictionary given a host_tree, a parasite_tree, and a reconciliation.
+        """
+        temporal_graph = ReconBuilder.build_temporal_graph(host_tree, parasite_tree, reconciliation)
+        ordering_dict = ReconBuilder.topological_order(temporal_graph)
+        return ordering_dict
+
+    @staticmethod
+    def check_topological_order(temporal_graph, ordering_dict):
+        """
+        Check the ordering_dict orders the nodes in temporal_graph in topological order
+        """
+        for node_tuple in temporal_graph:
+            for node_tuple_child in temporal_graph[node_tuple]:
+                is_child_leaf = node_tuple_child not in ordering_dict
+                # leaves are not ordered
+                if is_child_leaf:
+                    continue
+                assert(ordering_dict[node_tuple] < ordering_dict[node_tuple_child])
+
+    @classmethod
+    def test_detect_inconsistency_1(self):
+        """
+        test topological_order recognizes an inconsistent reconciliation
+        """
+        # a reconciliation with temporal inconsistency
+        host_tree = { 'hTop': ('Top', 'm0', ('m0', 'm1'), ('m0', 'm4')),
+        ('m0', 'm1'): ('m0', 'm1', ('m1', 'm2'), ('m1', 'm3')),
+        ('m0', 'm4'): ('m0', 'm4', ('m4', 'm5'), ('m4', 'm6')),
+        ('m1', 'm2'): ('m1', 'm2', None, None),
+        ('m1', 'm3'): ('m1', 'm3', None, None),
+        ('m4', 'm5'): ('m4', 'm5', None, None),
+        ('m4', 'm6'): ('m4', 'm6', None, None)}
+
+        parasite_tree = { 'pTop': ('Top', 'n0', ('n0', 'n1'), ('n0', 'n5')),
+        ('n0', 'n1'): ('n0', 'n1', ('n1', 'n2'), ('n1', 'n3')),
+        ('n1', 'n3'): ('n1', 'n3', ('n3', 'n4'), ('n3', 'n6')),
+        ('n1', 'n2'): ('n1', 'n2', None, None),
+        ('n3', 'n4'): ('n3', 'n4', None, None),
+        ('n0', 'n5'): ('n0', 'n5', None, None),
+        ('n3', 'n6'): ('n3', 'n6', None, None),}
+
+        reconciliation = { ('n0', 'm1'): [('T', ('n1', 'm1'), ('n5', 'm5'))],
+        ('n1', 'm1'): [('L', ('n1', 'm2'), (None, None))],
+        ('n1', 'm2'): [('T', ('n2', 'm2'), ('n3', 'm4'))],
+        ('n2', 'm2'): [('C', (None, None), (None, None))],
+        ('n4', 'm5'): [('C', (None, None), (None, None))],
+        ('n5', 'm5'): [('C', (None, None), (None, None))],
+        ('n6', 'm6'): [('C', (None, None), (None, None))],
+        ('n3', 'm4'): [('S', ('n4', 'm5'), ('n6', 'm6'))]}
+
+        ordering_dict = TestReconBuilder.build_ordering_dict(host_tree, parasite_tree, reconciliation)
+        assert(ordering_dict == None)
```

Comment:
use `self.assertTrue(...)`. Is there a `self.assertNone` or something like that? You should check :)